test/resource/resource.go

```diff
@@ -4,50 +4,48 @@ import (
 	"github.com/cloudskiff/driftctl/pkg/resource"
 	"github.com/cloudskiff/driftctl/test/schemas"
 	"github.com/hashicorp/terraform/providers"
-	"github.com/zclconf/go-cty/cty"
 )
 
 type FakeResource struct {
```

Comment:
Yeah we asked ourselves the same question. I think we could remove this (and maybe remove resource interface and only use abstractresource named as resource). But this will mean re writing all the test that use it and this is not easy for every test :/

Reply:
We could totally ditch the `FakeResource`

Reply:
We'll do this in another PR